### PR TITLE
travis: test Python bindings

### PR DESCRIPTION
**Pull request by matz-e**
We have so far omitted the Python bindings from the Travis tests.

### CODE REVIEW

**Comment by matz-e** [comment]
> LGTM. But I can't really evaluate the solution. If the tests are enough I let me know and will approve

I should maybe improve the ruby "grepper". But from the build log, tests are run, so this should be fine:
```
============================= test session starts ==============================

platform linux -- Python 3.6.8, pytest-5.2.2, py-1.8.0, pluggy-0.13.0

rootdir: /source, inifile: setup.cfg, testpaths: python/test

collected 49 items

python/test/test_mvd3.py ............................................... [ 95%]

..                                                                       [100%]

============================== 49 passed in 0.42s ==============================
```

File: setup.py [hunk 1]
```diff
@@ -57,7 +57,7 @@ def build_extension(self, ext):
             build_args += ["--", "/m"]
         else:
             cmake_args += ["-DCMAKE_BUILD_TYPE=" + cfg]
-            build_args += ["--", "-j{}".format(max(1, os.cpu_count() - 1))]
+            build_args += ["--", "-j{}".format(max(1, len(os.sched_getaffinity(0)) - 1))]
```
Comment (ferdonline) [inline setup.py]: Why do we override the defaults? There should be env vars controlling the nr of make/ninja processes, no?

File: setup.py [hunk 2]
```diff
@@ -57,7 +57,10 @@ def build_extension(self, ext):
             build_args += ["--", "/m"]
         else:
             cmake_args += ["-DCMAKE_BUILD_TYPE=" + cfg]
-            build_args += ["--", "-j{}".format(max(1, os.cpu_count() - 1))]
+            build_args += [
+                "--",
+                "-j{}".format(max(1, len(os.sched_getaffinity(0)) - 1))
```
Comment (ferdonline) [inline setup.py]: I think either we remove this or hardcode some small value. It's a small project, and cpu_count / affinity is not avail in old pythons

Comment (matz-e) [reply setup.py]: Good points. I'll update this tomorrow. Seems like I'm having some trouble with the right Docker image, too.

File: setup.py (hunk 1 above)
Comment (matz-e) [reply setup.py]: Not sure if there are dedicated variables controlling this.

**Pull request merged**